packages/protocol/contracts/tokenvault/BridgedERC1155.sol

```diff
@@ -77,12 +77,14 @@ contract BridgedERC1155 is
     function mint(
         address account,
         uint256 tokenId,
-        uint256 amount
+        uint256 amount,
+        string memory tokenUri
     )
         public
         onlyFromNamed("erc1155_vault")
     {
         _mint(account, tokenId, amount, "");
+        _uris[tokenId] = tokenUri;
```

Comment:
Dani, I still prefer not supporting URL at all. Firs of all, the data should be available on the source chain, secondly, this will make bridging cheaper and consistent.